src/libostree/ostree-repo-commit.c

```diff
@@ -424,6 +429,139 @@ add_size_index_to_metadata (OstreeRepo        *self,
 }
 
 static gboolean
+ostree_repo_commit_unpack_sizes (GVariant                    *entry,
+                                 OstreeContentSizeCacheEntry *sizes,
+                                 char                        *csum)
+{
+  gboolean ret = FALSE;
+  const guchar *buffer;
+  gsize bytes_read = 0;
+
+  gsize object_size = g_variant_get_size(entry);
```

Comment:
Space before paren. I also think it's more typical to keep all the declarations in the block at the top.